assets/prototype/domain/eventEditor/context/test/useDomTestData.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+import { eventId } from './data';
+
+const useDomTestData = () => {
+	// Set the DOM data
+	window.eeEditorEventData = { eventId };
+
+	// For Housekeeping
```

Comment:
Maybe add that this will do the cleanup on component unmount.
***
Side note:
I know that it is not recommended to think in the lifecycle methods, but it's hard to understand at first sight.